package.json

```diff
@@ -1,15 +1,16 @@
 {
   "name": "synctasks",
-  "version": "0.2.12",
+  "version": "0.2.13",
   "description": "An explicitly non-A+ Promise library that resolves promises synchronously",
   "author": "David de Regt <David.de.Regt@microsoft.com>",
   "scripts": {
     "test": "mocha dist/tests/synctaskstests.js",
     "webtest": "webpack",
-    "build": "tsc"
+    "build": "tslint --project tsconfig.json -r tslint.json --fix|| true && tsc"
```

Comment:
Consider having a separate 'tslint' task, then make "build": "npm run tslint || true && tsc" which is a little more readable. Just a thought.